src/main/java/twitterapp/src/TwitterAppResource.java

```diff
@@ -55,24 +58,24 @@ public Response getTimeline() {
     @Path("/tweet")
     public Response postTweet(String tweet){
     if (tweet.length() > MAX_LENGTH) {
-        System.out.println("Code 500: Tweet is too long, keep it within 280 characters");
+        log.error("Tweet is too long, keep it within 280 characters");
         return Response.serverError().entity("Tweet is too long, keep it within 280 characters").build();
     }
     else if(tweet.length() == 0){
-        System.out.println("Code 500: No tweet entered");
+        log.error("No tweet entered");
         return Response.serverError().entity("No tweet entered").build();
     }
     else {
         try {
             t.updateStatus(tweet);
         }
         catch (Exception e) {
-            System.out.println("Code 500: There was a problem on the server side, please try again later.");
+            log.error("There was a problem on the server side, please try again later.", e);
```

Comment:
same as above, who is looking at the error messages.  Nice job adding the exception